src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabServerConfig.java

```diff
@@ -0,0 +1,26 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import jakarta.inject.Inject;
+import org.jenkinsci.test.acceptance.po.Jenkins;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+public class GitLabServerConfig extends PageAreaImpl {
+
+    @Inject
+    public GitLabServerConfig(Jenkins jenkins) {
+        super(jenkins, "/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers");
+    }
+
+    public void configureServer(String url) {
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/name")).sendKeys("servername");
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).clear();
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/serverUrl")).sendKeys(url);
+
+        find(by.option("GitLab Personal Access Token")).click();
+
+        find(by.path("/io-jenkins-plugins-gitlabserverconfig-servers-GitLabServers/servers/validate-button")).click();
+
+        waitFor(driver, hasContent("Credentials verified for user"), 2);
```

Comment:
this should come in a validation area?  in which case - checking the validation area for the form over the content of HTML would be preferred.  see [`FormValidation`](https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/java/org/jenkinsci/test/acceptance/po/FormValidation.java) and if you use control objects then you can get this from the control.

Reply:
The FormValidation is for the entire Config page, I didn't find anything particular for server configuration